src/mcnntunes/app.py

```diff
@@ -450,30 +459,13 @@ def tune(self):
 
             # Make all plots needed in the report
             rep.plot_data(expdata, up, runs, best_x, display_output['summary'])
-            dof=len(expdata.y[0])
-            if runs.y_weight.any() != 1:
-                dof = np.sum(runs.y_weight != 0)
-            dof=dof-len(runs.params)
-            BestError = rep.plot_minimize(m, best_x, runs.scale_x(best_x), runs, self.config.use_weights, dof=dof)
+            rep.plot_minimize(m, best_x, runs.scale_x(best_x), best_std, runs, self.config.use_weights, dof=dof)
             if display_output['minimizer_type'] == 'CMAES':
                 rep.plot_CMAES_logger(m.get_fmin_output()[-3])
                 rep.plot_correlations(corr)
             display_output['avg_loss'] = rep.plot_model(nn.per_bin_nns, runs, expdata)
 
-            # Add best parameters
-            for i, p in enumerate(runs.params):
-                tmp=p.split("_")
```

Comment:
I was doing that only to remove the "Tune_Parameter_" in front of the parameter name, because the tabular is larger than the wrapper so removing it one don't have to scroll to see all the errors. Maybe there is a more elegant way
like: p=p.replace('Tune_Parameter_','')
but is only a style choice.

Reply:
Yes, I agree but this is not related to the error computation, which is the topic of this PR. If we want to remove the annotation tag, I believe it's better the remove it in ``yodaio.py`` during data loading.